### PR TITLE
v0.2.4 Improved Milling Feedback

### LINKED ISSUE #263: [UI] Improve feedback while milling
Instead of just "milling 2 stages", we should update this with the name of the stage, and progress.

### PR DESCRIPTION
**Pull request by patrickcleeve2**

### CODE REVIEW

**Review by patrickcleeve2 — COMMENTED** [review]
@Rohit-Kannachel  lgtm on demo
will test on microscope tomorrow

**Comment by patrickcleeve2** [comment]
@Rohit-Kannachel can you add the total time label and I will merge it

**Review by patrickcleeve2 — COMMENTED** [review]
@Rohit-Kannachel can you add the total time label and I will merge it

File: fibsem/ui/FibsemMillingWidget.py [hunk 1]
```diff
@@ -627,26 +646,88 @@ def _toggle_interactions(self, enabled: bool = True, caller: str = None, milling
...
+
+        info = [idx, total, est_time]
+
+        self.progress_bar_worker = self.start_progress_bar(info)
+        self.progress_bar_worker.finished.connect(self.finish_progress_bar)
+
+        self.progress_bar_worker.start()
+
+    @thread_worker
+    def start_progress_bar(self,info):
```
Comment (patrickcleeve2) [inline fibsem/ui/FibsemMillingWidget.py:680]: @Rohit-Kannachel  change info to a dictionary

Comment (Rohit-Kannachel) [reply fibsem/ui/FibsemMillingWidget.py:680]: done

**Pull request merged**